kvmagent/kvmagent/plugins/zbs_storage_plugin.py

```
@@ -39,6 +275,65 @@ def check_host_storage_connection(self, req):
 
+    @kvmagent.replyerror
+    @bash.in_bash
+    def luks_clone(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
+        rsp = ZbsLuksRsp()
+        try:
+            self._clone_plain_to_luks(cmd.srcPath, cmd.dstPath,
+                                      getattr(cmd, 'encryptedDek', None),
+                                      getattr(cmd, 'virtualSizeForLuksClone', None))
+            rsp.actualSize = self._raw_cbd_actual_size(cmd.dstPath)
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)